machine/wnc_x86_64/wnc.patch/installer/x86_64/install-arch

```diff
@@ -0,0 +1,499 @@
+# x86_64 specific ONIE installer functions
```

Comment:
What kind of board is this for?  Is this going to be a shipping product?

For a shipping product I don't think you want to store the TLV data on the hard disk.  Too fragile.

For a development board or eval board it is an interesting idea.  The virtual machine implementation might also be able to use this approach.  Interesting.

If the board in question has some MTD flash then the EEPROM data can be stored there.  Several platforms use this approach.  The busybox EEPROM code already has support for doing that.  See these Accton machines:
- https://github.com/opencomputeproject/onie/blob/master/machine/accton/accton_as4600_54t/busybox/conf/config
- https://github.com/opencomputeproject/onie/blob/master/machine/accton/accton_as5600_52x/busybox/conf/config
- https://github.com/opencomputeproject/onie/blob/master/machine/accton/accton_as5610_52x/busybox/conf/config

Also for development / eval purpose it is entirely legitimate to just skip the EEPROM implementation.  The KVM virtual machine does not implement the EEPROM for example.  See the machine.make for that machine where `I2CTOOLS_ENABLE = no` is set.  Also that machine defines a fake implementation of onie-sysinfo for serial # and MAC address.  See the overrides here:
https://github.com/opencomputeproject/onie/blob/master/machine/kvm_x86_64/rootconf/sysroot-lib-onie/sysinfo-platform

Regarding the certification questions:

1.  if the board in question is only capable of 57600 then that will be OK.  It is a bit of a odd constraint though.

2.  your pull request will only affect the board in the question.  It will not affect other boards, so that should be OK.